src/main/webapp/app/components/newCompanyForm/NewCompanyForm.tsx

```diff
@@ -31,8 +34,20 @@ import { AdditionalInfoSelect } from 'app/shared/dropdown/AdditionalInfoSelect';
 import {
   debouncedCompanyNameValidator,
   fieldRequiredValidation,
-  textValidation,
 } from 'app/shared/utils/FormValidationUtils';
+import CompanyAdditionalInfo from 'app/pages/companyPage/CompanyAdditionalInfo';
+
+const defaultInfo: CompanyAdditionalInfoDTO = {
+  license: {
+    autoRenewal: false,
+    activation: '',
+    termination: {
+      date: '',
+      notificationDays: 60,
```

Comment:
30

Reply:
auto-renew true

Reply:
add warning to user that email will be sent if inside of notification window when the user updates the company page.